### ISSUE TITLE
Airmon-ng update

### ISSUE THREAD

**Issue by Dutzzu** [vk496/linset#9]
Hello,

I don't know if you know about this update, but I think it cause problems to the script: https://www.kali.org/penetration-testing/pixiewps-reaver-aircrack-ng-updates/

The basic idea is that the new naming convention of the wireless virtual interfaces – wlanXmon, as opposed to monX. 

So, it does not find any cards and gives the following error: 
"No se han encontrado tarjetas Wireless. Cerrando..."

Thank you!

**Comment by frafrichile** [comment]
I am having the same issue.  
I think you are right. The name convention changed from monX to wlanXmon

Thank you !

**Comment by Dutzzu** [comment]
I think that's not suck a big problem. With an update of the script it should work just fine. :-)

**Comment by dotai2012** [comment]
@Dutzzu Please help me how to solve this prolem :((

**Comment by difol** [comment]
All check the pull request for support Airmon-ng

**Comment by digit8eight** [comment]
Please update linset to work with latest airmon-ng changes.

**Comment by joepastor** [comment]
The problem seems to be the difference between mon0 and wlan0mon that uses the latest version of aircrack. 
Does anybody have the solution for this?

**Comment by aididhaiqal** [comment]
Yep,I have solution (Working linset) but i still fixing it.To many bug

**Comment by joepastor** [comment]
Please let me know if you need help. I'm available for testing .
Thank you
Joe

**Comment by aididhaiqal** [comment]
joebeach,I will release a beta soon(English Version and fixed some things that the writer do)

**Comment by aididhaiqal** [comment]
Solved a very big bug :)
Update:Just test it and found tempory solution,Will release after few more bug solved

**Comment by aididhaiqal** [comment]
Ok,I have fix bug until the web part,It seem i need to fix much line :(.If it not done today,It will this week

**Comment by jazzy17** [comment]
You can also ask me for testing, i m looking to hack wpa2 without using bruteforce and and two wireless devices as i have only one wireless device which can be put into monitor mode and i dont wanna buy another, and all the other evil twin attacks use two wireless cards. can u verify that it uses only one wifi card. THANKS

**Comment by aididhaiqal** [comment]
![screenshot from 2015-09-10 11 29 48](https://cloud.githubusercontent.com/assets/10329190/9779551/50d2054e-57af-11e5-9525-a9502277e752.png)
![screenshot from 2015-09-10 11 29 49](https://cloud.githubusercontent.com/assets/10329190/9779552/51179154-57af-11e5-9d57-1c45a03c53a5.png)
More to come soon

**Comment by jazzy17** [comment]
@aidid12 when are you gonna make it available for us, waiting!!!!!!

**Comment by frafrichile** [comment]
I have a Realtek RTL8187L !

**Comment by aididhaiqal** [comment]
Soon and i think to join vk496 developing

```
 On Thursday, September 10, 2015 9:01 PM, frafrichile <notifications@github.com> wrote:
```

 I have a Realtek RTL8187L ! —

**Comment by nworks16** [comment]
i have a linset version Fixed to work on Kali linux 2.0 and parrot 2.0 if u want try named "WOLFTOOL"

**Comment by wermaxta** [comment]
how to start linset on Kali linux 2.0 thanks to DD0s
https://www.youtube.com/watch?v=sGl9iiJWr3c

**Comment by aididhaiqal** [comment]
Not realy work actualy,it just fix airmon issue but not others

Sent from Yahoo Mail on Android

**Comment by joepastor** [comment]
I tried it and it worked fine for me.
Yes, there are some minor bugs (or I think..) to solve, but the interface
issue is fixed.

Well done!

2015-09-13 20:12 GMT-03:00 aidid12 notifications@github.com:

> Not realy work actualy,it just fix airmon issue but not others
> 
> Sent from Yahoo Mail on Android
> 
> From:"wermaxta" notifications@github.com
> Date:Sun, 13 Sep, 2015 at 11:44 pm
> Subject:Re: [linset] Airmon-ng update (#9)
> 
> how to start linset on Kali linux 2.0 thanks to DD0s
> https://www.youtube.com/watch?v=sGl9iiJWr3c
> 
> —
> Reply to this email directly or view it on GitHub.￼
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/vk496/linset/issues/9#issuecomment-139927608.

## 

José Pastor López
(15) 4072-7071
jlopez.mail@gmail.com
Skype: jlopez.mail

**Comment by aididhaiqal** [comment]
Yep interface issue is easy actualy

Sent from Yahoo Mail on Android

**Comment by shareourthought** [comment]
i got a solution for that problem try the blow one its patched with airmon-ng.

Download Link : http://www13.zippyshare.com/v/YE3gov7U/file.html

Instructions : 
1.Download that Linset patched and extract it
2.Change the terminal directory to extracted folder 
3.Type this command in terminal "chmod +x linset" and "chmod +x airmon"
4.Finally "./linset"

For better support follow me on :

Youtube : https://www.youtube.com/channel/UCfmEvfxFowO1anbbLuFi3BA

Blogger : https://shareoursthought.blogspot.com

Facebook : https://www.facebook.com/naninavveen

G-mail : shareourthought@gmail.com

Google+ : https://plus.google.com/u/0/+NaveenPo...

Twitter : https://twitter.com/shareourthought

**Comment by frafrichile** [comment]
This is what I get 
wc: /tmp/TMPlinset/dump

-01.csv: No such file or directory
./linset: line 977: [: -le: unary operator expected
cat: /tmp/TMPlinset/dump-01.csv: No such file or directory
expr: syntax error
                         List of APs Objective   

 #      MAC                      CH      SEC      PWR    ESSID

grep: /tmp/TMPlinset/dump-01.csv:: No such file or directory
grep: invalid: No such file or directory
grep: number: No such file or directory
grep: of: No such file or directory
grep: lines: No such file or directory
 1) head: /tmp/TMPlinset/dump-01.csv: invalid number of lines     100% 

how can I fix?﻿

**Comment by SagastyCodex** [comment]
i tried some fixed linset in my Kali 2.0 but in last part of Option on where i chooses "IDIOMA" whenever i entered 1 it freezes my Entire system.... so no choices left but Forced shutdown anyone has fix with dis???

**Comment by shareourthought** [comment]
Dear kerjboy,

Give me the linset script your are using and prove some screenshots for
more information.

*Note: *I wrote a comment in linset repository in git-hub. go through that
procedure it will helpful.

**Comment by SagastyCodex** [comment]
i have made a video since my entire system is Freezed i couldn't take some screenshots. later i will upload it

**Comment by strasharo** [comment]
@frafrichile , are you using linset inside a X server session or you're trying to run in terminal without X? I think that this is the error that you get when you try to run linset without X.

**Comment by SagastyCodex** [comment]
hi @shareourthought here's the video i made plsss take a look sorry for bad resolution i only used my old iphone 4s.  https://youtu.be/k9O5jex2hbU
